src/CreateKeyInterface.php

```diff
@@ -11,6 +11,8 @@
  * create request, instead of creating a new object.
  *
  * @see https://docs.theplatform.com/help/wsf-creating-or-updating-data-objects-using-create-keys
+ *
+ * @todo Move to the annotation and convert to CustomKeyInterface.
```

Comment:
I may remove this entirely, as two of the three methods are covered by the annotation. However, I'm unsure yet if developers will be able to statically define custom keys in a configuration somewhere, or if it needs to be runtime code.